ogc/browse-ogc-api-feature-service/src/main/java/com/esri/samples/browse_ogc_api_feature_service/BrowseOgcApiFeatureServiceSample.java

```diff
@@ -0,0 +1,292 @@
+
+          // hide progress indicator
+          progressIndicator.setVisible(false);
+
+          // get the service metadata and then a list of available collections
+          OgcFeatureServiceInfo serviceInfo = ogcFeatureService.getServiceInfo();
+          List<OgcFeatureCollectionInfo> layerList = serviceInfo.getFeatureCollectionInfos();
+          ogcLayerNamesListView.getItems().addAll(layerList);
+
+          // populate the list view with layer names
+          ogcLayerNamesListView.setCellFactory(list -> new ListCell<>() {
+
+            @Override
+            protected void updateItem(OgcFeatureCollectionInfo ogcFeatureCollectionInfo, boolean bln) {
+              super.updateItem(ogcFeatureCollectionInfo, bln);
+              if (ogcFeatureCollectionInfo != null) {
+                String titleOfOgcFeatureServiceLayer = ogcFeatureCollectionInfo.getTitle();
+                setText(titleOfOgcFeatureServiceLayer);
+              }
+            }
+          });
+          // when a layer is selected, enable the load selected layer button
+          ogcLayerNamesListView.setOnMouseClicked(e -> loadSelectedLayerButton.setDisable(false));
+
+          // if the feature service fails to load, display an error
+        } else {
+          new Alert(Alert.AlertType.ERROR, "Failed to load OGC feature service: " +
+            ogcFeatureService.getLoadError().getCause().getLocalizedMessage()).show();
+        }
+      });
+
+      // if the text field is blank and the load button is clicked, display an error
+    } else {
+      new Alert(Alert.AlertType.ERROR, "Enter a service url to continue").show();
+    }
+  }
+
+  /**
+   * Adds a OGCFeatureCollectionInfo to the map's operational layers.
+   *
+   * @param ogcFeatureCollectionInfo the ogcFeatureCollectionInfo that the map will display
+   */
+  private void updateMap(OgcFeatureCollectionInfo ogcFeatureCollectionInfo) {
+
+    progressIndicator.setVisible(true);
+
+    // clear the map's operational layers
+    map.getOperationalLayers().clear();
+
+    // create an OGC feature collection table from the feature collection info
+    var ogcFeatureCollectionTable = new OgcFeatureCollectionTable(ogcFeatureCollectionInfo);
+
+    // set the feature request mode to manual
+    // in this mode, the table must be manually populated as panning and zooming won't request features automatically.
+    ogcFeatureCollectionTable.setFeatureRequestMode(ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE);
+
+    // populate the table and then remove progress indicator and set the viewpoint to that of the layer's full extent when done
+    var queryParameters = new QueryParameters();
+    // set a limit of 1000 on the number of returned features per request, the default on some services could be as low as 10
+    queryParameters.setMaxFeatures(1000);
+
+    try {
+      // populate the table with the query, leaving existing table entries intact, setting the outfields parameter to null requests all fields
```

Comment:
```suggestion
      // populate the table with the query parameters
      // set the clearCache parameter to false to include existing table entries, and set the outfields parameter to null to request all fields
```
Just a suggestion, feel free to reword it just didn't quite read right.